docs/UserGuide.adoc

```diff
@@ -209,79 +172,104 @@ Format: `find n\[name]`
+`list c\[classmate/teammate]` or `find` command,
 
-Sets the profile picture of the second contact in your address book to aloha.png.
+`edit i\1 p\88888888`
 
-==== Delete Picture in Contact: `deletepic`
-You can delete the profile picture of a contact in your address book.
+NOVA will edit the phone number of the first contact in your address book to 88888888.
 
-Format: `deletepic i\[index]`
+==== Add Category Specific Remarks: `remark`
+You can add remarks that are category specific, to a contact.
+
+Format: `remark i\[index] r\[remark]`
 
 [NOTE]
 ====
-* Deletes the picture at the specified `[index]` in the displayed person list
+* Adds category specific remark at the specified `[index]` in the displayed contact list
 * The `[index]` must be a positive integer. (E.g. 1, 2, 3, ...)
+* To remove any remarks that you have added for a contact, use `remark i\[index] r\`
 ====
 
 Example: +
-`deletepic i\2`
+Suppose you want add remark to the first contact in your address book after using `list`,
+`list c\[classmate/teammate]` or `find` command,
 
-Deletes the profile picture of the second contact in your address book.
+`remark i\1 r\He's a nice teammate`
 
-==== Add Category Specific Remarks: `remark`
-You can add remarks that are category specific, to a contact.
+NOVA will add the remark "He's a nice teammate" to the first contact in your address book.
 
-Format: `remark i\[index] r\[remark]`
+==== Delete Contact: `delete`
+You can delete a contact that you have added. If the contact you try to delete does not exist, NOVA will let you know.
+
+Format: `delete i\[index]`
 
 [NOTE]
 ====
-* Adds category specific remark at the specified `[index]` in the displayed person list
+* Deletes the contact at the specified `[index]` in the displayed contact list
 * The `[index]` must be a positive integer. (E.g. 1, 2, 3, ...)
 ====
 
 Example: +
-`remark i\2 r\He's a nice teammate`
+Suppose you want to delete the first contact in your address book after using `list`,
+`list c\[classmate/teammate]` or `find` command,
+
+`delete i\1`
 
-Adds the remark "He's a nice teammate" to the second contact in your address book.
+NOVA will delete the first contact in your address book.
+
+==== Clear all Contacts: `clear`
+You can clear all the contacts that you have added in your address book.
+
+Format: `clear`
+
+[WARNING]
+Using `clear` command will **delete all** the contacts that you have saved.
 
 ==== Undo Command: `undo`
-You can undo a command that you have entered.
+You can undo a command that you have entered. If you cannot undo, NOVA will let you know.
 
 Format: `undo`
 
 ==== Redo Command: `redo`
 
-You can redo undone commands.
+You can redo undone commands. To use `redo`, you must first have used `undo`. If you cannot redo, NOVA will let you know.
 
 Format: `redo`
 
 [WARNING]
 After you successfully enter a new command, you will lose all the undone commands.
```

Comment:
Help me modify this warning a little.
"After you successfully entered a command (apart from undo) after an undo command, you will lose all undone commands."